### PR TITLE
Kp cleanup

### PR DESCRIPTION
**Pull request by interviewperfect**

### CODE REVIEW

File: frontend/controllers/PlaceController.js [hunk 1]
```diff
@@ -26,12 +23,3 @@ exports.postAPIPixel = (req, res, next) => {
     }
     paintWithUser(req.user);
 };
-
-exports.getAPITimer = (req, res, next) => {
```
Comment (EricRabil) [inline frontend/controllers/PlaceController.js:30]: no

File: frontend/controllers/PlaceController.js [hunk 2]
```diff
@@ -7,17 +7,14 @@ exports.postAPIPixel = (req, res, next) => {
...
     function paintWithUser(user) {
-        if (!user.canPlace(req.place)) return res.status(429).json({ success: false, error: { message: "You cannot place yet.", code: "slow_down" } });
         if (!req.body.x || !req.body.y || !req.body.hex) return res.status(400).json({ success: false, error: { message: "You need to include all paramaters", code: "invalid_parameters" } });
         var x = Number.parseInt(req.body.x), y = Number.parseInt(req.body.y);
         if(Number.isNaN(x) || Number.isNaN(y)) return res.status(400).json({ success: false, error: { message: "Your coordinates were incorrectly formatted", code: "invalid_parameters" } });
         if (!user.canPlaceColour(req.body.hex, req.place)) return res.status(400).json({ success: false, error: { message: "You can't place that colour.", code: "disallowed_colour" } });
         var rgb = req.place.paintingManager.getColourRGB(req.body.hex);
         req.place.paintingManager.doPaint(rgb, x, y, user).then((pixel) => {
             return User.findById(user.id).then((user) => {
-                var seconds = user.getPlaceSecondsRemaining(req.place);
```
Comment (EricRabil) [inline frontend/controllers/PlaceController.js:18]: no

File: frontend/models/user.js [hunk 3]
```diff
@@ -293,24 +293,6 @@ UserSchema.methods.addPixel = function(colour, x, y, app, callback) {
     });
 }
 
-UserSchema.methods.getPlaceSecondsRemaining = function(app) {
```
Comment (EricRabil) [inline frontend/models/user.js:296]: no

File: frontend/routes/api.js [hunk 4]
```diff
@@ -103,8 +103,6 @@ function APIRouter(app) {
     
     router.post("/place", requireUser, PlaceController.postAPIPixel);
 
-    router.get("/timer", requireUser, PlaceController.getAPITimer);
```
Comment (EricRabil) [inline frontend/routes/api.js:106]: no

File: frontend/views/public/views/place.pug [hunk 5]
```diff
@@ -45,18 +42,10 @@
                             (#[span], #[span])
             .ctn-row
                 .content#palette
-                    #menu-content-ctn
```
Comment (EricRabil) [inline frontend/views/public/views/place.pug:48]: why did you remove templates and warps they're literally implemented

File: solidity/cryptoplace.sol [hunk 6]
```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.19;
```
Comment (EricRabil) [inline solidity/cryptoplace.sol:1]: Please explain

**Review by EricRabil — COMMENTED** [review]
Why did you move everything to `frontend`? Is this a satirical PR?

**Comment by AppleBetas** [comment]
Idea: keep your own forks with customizations that will only benefit you separate, rather than trying to merge them back into the base.
